recipes/dtc/meta.yaml

```diff
@@ -0,0 +1,166 @@
+
+source:
+  url: https://git.kernel.org/pub/scm/utils/dtc/dtc.git/snapshot/dtc-{{ version }}.tar.gz
+  sha256: 6401c9a0f577a270df4632bf0f3e5454ccc7a5ca3caefa67a3e1c29c9c6b8c60
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - bison
+    - flex
+    - make
+    - swig
+    - pkg-config
+  host:
+    - python {{ python }}
+    - yaml
+    - valgrind
+  run:
+    - python {{ python }}
+
+outputs:
+  - name: dtc
+    script: install.sh
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - make
+      host:
+        - {{ pin_subpackage("libfdt", exact=True) }}
+        - yaml
+        - valgrind
+      run:
+        - {{ pin_subpackage("libfdt", max_pin="x.x") }}
+    test:
+      requires:
+        - {{ compiler('c') }}
+        - make
+      source_files:
+        - util.c 
+        - util.h
+        - version_gen.h
+        - tests
+      commands:
+        - convert-dtsv0 --help
+        - dtc --help
+          # dtdiff is a shell script that can pipe two device-tree files through dtc and diff them
+          # there are no specific tests for it in the dtc testsuite and the diff isn't completely
+          # clean even if you run 'dtdiff tests/test_tree1.{dts,dtb}' because the representation
+          # of various data is formatted slightly differently in the unpacked dtb
+        - 'test -x "$PREFIX"/bin/dtdiff'
+        - fdtdump --help
+        - fdtget --help
+        - fdtoverlay --help
+        - fdtput --help
+        # There is only one compilation unit needed from the toplevel project. Running the tests
+        # from inside tests subdir removes the binaries and library target from the prereqs and
+        # that's what we want for testing the packaged artifacts. So, we copy three tiny files
+        # from the toplevel project into tests and then we can 'cleanly' rerun the tests using the
+        # packaged artifacts
+        - cp util.c util.h version_gen.h tests
+        # use Makefile.tests directly so that binaries won't be built but used from installed package
+        # NO_PYTHON because we test that in pylibfdt
+        # VECHO=: to enable Makefile.tests to stand on it's own
+        # LDLIBS=-lfdt to correctly find installed libfdt instead of having in-tree library an explicit dependency listed in the link command
+        # LIBDL= needs to also include -lfdt because of how it is used in rules to override the  builtin and not use LDLIBS
+        # TESTS_PREFIX=./ to work around 'cd $TEST_PREFIX' in check recipe
+        # TEST_BINDIR="$PREFIX/bin" because testutils.sh hardcodes the paths to binaries, defaulting to ../
+        - 'NO_PYTHON=1 TEST_BINDIR="$PREFIX/bin" make -C tests -f Makefile.tests VECHO=: LDLIBS=-lfdt tests_clean check TESTS_PREFIX=./ LIBDL="-lfdt -ldl"'  # [linux]
+        - 'NO_PYTHON=1 TEST_BINDIR="$PREFIX/bin" make -C tests -f Makefile.tests VECHO=: LDLIBS=-lfdt tests_clean check TESTS_PREFIX=./ LIBDL=-lfdt'  # [osx]
```

Comment:
Could you explain what this test is for? It looks like you already tested all of the binaries in previous tests (either by running help or checking for their existence).